rma_account/models/rma_order_line.py

```diff
@@ -129,21 +128,17 @@ def _onchange_product_id(self):
             res["domain"] = {}
         if not self.product_id:
             domain = [
-                "&",
                 "|",
                 ("move_id.partner_id", "=", self.partner_id.id),
                 ("move_id.partner_id", "child_of", self.partner_id.id),
-                ("exclude_from_invoice_tab", "=", False),
```

Comment:
Replace by `("display_type", "=", "product"),` to keep the same behavior ?

Reply:
Done!